src/coreclr/jit/importercalls.cpp

```diff
@@ -2118,7 +2137,7 @@ void Compiler::impPopArgsForSwiftCall(GenTreeCall* call, CORINFO_SIG_INFO* sig,
 
     // If using SwiftError*, spill entire stack as we will need to reuse the
     // error argument after the call.
-    if (checkEntireStack)
+    if (spillStack)
```

Comment:
Could you update this comment to mention the SwiftIndirectResult case, please?

Reply:
I moved the existing comment up to where we assign `spillStack` in the `SwiftError*` case instead (there is already a comment on the new `spillStack = true` I added)